tools/travis/deploy_docs.sh

```diff
@@ -19,7 +19,10 @@ then
 
     git rm -r dev
     cp -r ../doc/build/html dev
+    mv dev/binder .
+    cp ../binder/postBuild binder/
```

Comment:
Can we change the paths with a name containing sphinx-gallery. I'm afraid that soon, i'll wonder what's binder.

Reply:
My remark also applies to "requirements/binder"